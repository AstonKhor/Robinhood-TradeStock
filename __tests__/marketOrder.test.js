/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import MarketOrder from '../client/components/marketOrder.jsx';


describe('<MarketOrder /> component', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<MarketOrder />);

    expect(tree).toMatchSnapshot();
  });

  it('initiates with state of 0 shares', () => {
    const wrapper = shallow(<MarketOrder />);
    expect(wrapper.state().shares).toBe(0);
  });
});
