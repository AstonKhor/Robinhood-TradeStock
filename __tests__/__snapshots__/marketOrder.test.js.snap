// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MarketOrder /> component matches the snapshot 1`] = `
Array [
  <div
    className="sc-bxivhb cyWHkv"
  >
    <label
      className="sc-bwzfXH dbwzZe"
    >
      Shares
    </label>
    <input
      className="sc-bdVaJa eqHRdS"
      onChange={[Function]}
      placeholder="0"
      type="number"
    />
  </div>,
  <div
    className="sc-bxivhb cyWHkv"
  >
    <span
      className="sc-ifAKCX bNpiVu"
    >
      Market Price
    </span>
    <span
      className="sc-EHOje cANiii"
    >
      $
    </span>
  </div>,
]
`;
